src/Pixel.Identity.UI.Client/Shared/MainLayout.razor

```diff
@@ -10,10 +12,10 @@
             <MudIconButton Icon="@Icons.Material.Outlined.Menu" Color="Color.Inherit" OnClick="@((e) => DrawerToggle())" />
         </MudToolBar>
         <MudHidden Breakpoint="Breakpoint.Xs">
-            <MudText Typo="Typo.h6" Class="ml-4">Pixel Identity</MudText>
+            <MudText Typo="Typo.h6" Class="ml-4">@_brand.Name</MudText>
```

Comment:
Wondering if it is possible to separate header in to a component and allow users to setup a custom one which can be loaded dynamically at runtime (e.g. by mounting a path on container). Also, the logos are not plugged in yet and would require them to be hosted somewhere else.
I will try to explore this a little.